src/collector/StatsParser.h

```diff
@@ -43,12 +52,21 @@ class StatsParser : public Parser
     NodeStat & get_node_stat()
     { return m_data.node; }
 
+    std::map<unsigned int, uint64_t> & get_rofs_errors()
+    { return m_rofs_errors; }
+
 public:
+    virtual bool Key(const char* str, rapidjson::SizeType length, bool copy);
+
     virtual bool EndObject(rapidjson::SizeType nr_members);
 
+    virtual bool Bool(bool b)
```

Comment:
:)
It is the same line you commented in another pull request. Fixed: https://github.com/yandex/mastermind/commit/1db53ccc62975de753a6fa8deb5613d61b453065